### DISCUSSION TITLE
If I don't need i18n pathnames, how can I implement a generic Link?

### DISCUSSION THREAD

**Discussion by aboutjquery** [amannn/next-intl#1292] [Q&A]
I recently saw the `example-app-router` demonstration updated, so I downloaded it to test it out. Now there are two languages, `en` and `zh`.

When `config.ts` is

```
export const pathnames = {
 '/': '/',
 '/products': '/products',
} satisfies Pathnames<typeof locales>;
```

To display I don't need multilingual `pathnames`, I just need something like `en/products/` or `zh/products/`

When I use in the page

```
import { Link } from '@/navigation';
<Link href="/contact">Contact</Link>
```

I found two problems

No. 1, for example, the current page is `zh/about/`
Clicking on the above link sometimes jumps to `en/about/` and sometimes it works fine.

Second, you will get an error when using `npm run build`
`Type error: Type ''/${string}`' is not assignable to type '"/" | "/products"`

I believe it’s because I didn’t add `/contact` to the `pathnames` of `config.ts`

Pages often have many dynamic routes. If adding `pathnames` one by one is very troublesome, what should I use to make general links? I just hope to keep `{locale}` operating normally.

Thanks

**Comment by emirxmertoglu** [comment]
I have similar problem any news?
